### DISCUSSION TITLE
Hello everyone

### DISCUSSION THREAD

**Discussion by aabdurakhmanov** [AAbdurakhmanov/TTS_NLP_python_graduate_work#3] [Q&A] [answered]
Hi everyone plz give me best solution for my simple projects thanks for all

**Comment by FrankBenjamen** [comment, marked as answer]
Hi  bro your project is best only learning words based on artificial intelligence would be a good help

**Comment by FrankBenjamen** [comment]
And other ways you need more words and voices

**Comment by BahtiyourR2** [comment]
I agree with that
